es2csv.py

```diff
@@ -267,6 +267,11 @@ def main():
     p.add_argument('-e', '--meta_fields', dest='meta_fields', action='store_true', help='Add meta-fields in output.')
     p.add_argument('-v', '--version', action='version', version='%(prog)s ' + __version__, help='Show version and exit.')
     p.add_argument('--debug', dest='debug_mode', action='store_true', help='Debug mode on.')
+    p.add_argument('--verify-certs', dest='verify_certs', action='store_true', help='Verify SSL Certs (default is False)')
+    p.add_argument('--ca-certs', dest='ca_certs', default='', type=str, help='Location of CA cert file for Cert verification')
```

Comment:
ca_cert, client_cert and client_key by default are None, https://github.com/elastic/elasticsearch-py/blob/5.0/elasticsearch/connection/http_requests.py#L30
It is better to stay with None then to pass empty string